engine/src/main/java/com/datatorrent/stram/client/RecordingsAgent.java

```diff
@@ -640,7 +640,7 @@ public JSONObject process(WebResource.Builder webResource, Class<JSONObject> cla
           }
       );
     } catch (Exception ex) {
-      LOG.error("Exception caught", ex);
+      LOG.error("Cannot start recording: ", ex);
```

Comment:
@davidyan74 : Should logging be not done at same levels? I see here it is "error" and earlier it is done at "warn" and also this will print stacktrace but log messages at 546 line won't...

Any reason for inconsistent log levels?